source/extensions/resource_monitors/downstream_connections/BUILD

```diff
@@ -25,6 +25,7 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    visibility = ["//visibility:public"],
```

Comment:
context behind why we need this?

Reply:
Without it i get below error message when running cx_limit_overload_integration_test:
```
ERROR: /home/nezdolik/envoy/test/integration/BUILD:2164:14: in cc_test rule //test/integration:cx_limit_overload_integration_test: target '//source/extensions/resource_monitors/downstream_connections:config' is not visible from target '//test/integration:cx_limit_overload_integration_test'.
```

Reply:
I think we could also get away with:

```
 extra_visibility = [
       "//test/integration:__subpackages__",
  ],
```

I wonder if it'd work if the test was in the `test/extensions/resource_monitors/downstream_connections/cx_limit_overload_integration_test` or similar. e.g. I see the similar definitions in `source/extensions/resource_monitors/injected_resource` and it works for `test/extensions/resource_monitors/injected_resource` without additional changes to visibility.